scripts/common/preflights.sh

```diff
@@ -30,10 +30,15 @@ function init_preflights() {
 # if kurl pods like ekco not be running then we should bail
 function bail_if_kurl_pods_are_unhealthy() {
     if commandExists kubectl; then
-      log "Awaiting 2 minutes to check kURL Pod(s) are Running"
-      if ! spinner_until 120 check_for_running_pods kurl; then
-          bail "Kurl has unhealthy Pod(s). Check the namespace kurl. Restarting the pod may fix the issue."
-      fi
+        # if host preflight are being ignored we just move forward.
```

Comment:
Should not it be in another PR so that we can either generate the release notes accordingly to know what is fixed or not in each version which makes easier when we need to gathering this info in the support

> Fixes pods check to allow ignore it failures when HOST_PREFLIGHT_IGNORE flag is used

Reply:
For sake of simplicity I am keeping this here.